config/standard/contact-summary-extras.js

```diff
@@ -429,3 +429,49 @@ function getFollowupExitReport(){
     return r.form === 'nutrition_followup' && r.fields.task && r.fields.task === 'exit';
   });
 }
+
+module.exports = {
```

Comment:
We could probably use the short form to reduce boilerplate

```javascript
module.exports = {
  now,
  pregnancyForms
 ...
}
```

Reply:
That's ES6 syntax :(  Working on it, but didn't make it in this round.